specification/appconfiguration/data-plane/Microsoft.AppConfiguration/preview/2022-11-01-preview/appconfiguration.json

```diff
@@ -1723,6 +1767,37 @@
       },
       "readOnly": true
     },
+    "OperationDetails": {
+      "properties": {
+        "id": {
+          "description": "The unique id of the operation.",
+          "type": "string"
+        },
+        "status": {
+          "description": "The current status of the operation",
+          "enum": [
+            "NotStarted",
+            "Running",
+            "Succeeded",
+            "Failed",
+            "Canceled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "modelAsString": false,
+            "name": "State"
+          }
+        },
+        "error": {
+          "description": "An error, available when the status is `Failed`, describing why the operation failed.",
+          "$ref": "#/definitions/Error"          
```

Comment:
This isn't the correct error schema for LRO.

Reply:
Should be the ErrorDetail object mentioned [here](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#handling-errors)

Reply:
@HarshaNalluru I have updated the spec to use the [ErrorDetail schema](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#handling-errors) defined in the Azure guidelines. could you take another look?

Reply:
Looks good @jimmyca15, thank you!